test-case/verify-firmware-presence.sh

```diff
@@ -18,7 +18,7 @@ source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
 
 func_opt_parse_option $*
 
-path="/lib/firmware/intel/sof"
+path="/lib/firmware/intel/sof/community"
```

Comment:
Not all platform is using this path, please update this part to change the path with DMI info to follow the kernel change.